js/src/elements/form-behavior.ts

```diff
@@ -519,10 +531,10 @@ export class FormBehaviorElement extends HTMLElement {
 		const label = getLabel(input);
 		const group = getField(input);
 
-		if (displayValidity) {
-			setValidationMessage(input, '');
-			group.classList.remove('errored');
-		}
+		// if (displayValidity) {
+		// 	setValidationMessage(input, '');
+		// 	group.classList.remove('errored');
+		// }
```

Comment:
Commented out this block as it was resetting on the checkbox group error notes.  Lets test that is not actually needed.

Reply:
I think we should try to get this one right (ensure it isn't needed and remove the comment, or determine it is needed and uncomment) before we commit this change

Reply:
I tested it several times and all seems to operate.  What I am asking is for each of you to do the same.

I do not intend to close this PR with it remaining.

Reply:
Removed as my testing was not affected by removing it.  (or I am missing something completely)